code/modules/cargo/exports/sheets.dm

```diff
@@ -11,130 +7,102 @@
 
-// Cat hide. Just in case Runtime is catsploding again.
 /datum/export/stack/skin/cat
-	cost = 2000
+	cost = 150
 	contraband = TRUE
 	unit_name = "cat hide"
 	export_types = list(/obj/item/stack/sheet/animalhide/cat)
 
-// Corgi hide. You monster.
 /datum/export/stack/skin/corgi
-	cost = 2500
+	cost = 200
 	contraband = TRUE
 	unit_name = "corgi hide"
 	export_types = list(/obj/item/stack/sheet/animalhide/corgi)
 
-// Lizard hide. Very expensive.
 /datum/export/stack/skin/lizard
-	cost = 5000
+	cost = 150
 	unit_name = "lizard hide"
 	export_types = list(/obj/item/stack/sheet/animalhide/lizard)
 
-// Gondola hide. Mindbogglingly expensive.
 /datum/export/stack/skin/gondola
-	cost = 10000
+	cost = 500
 	unit_name = "gondola hide"
 	export_types = list(/obj/item/stack/sheet/animalhide/gondola)
 
-// Alien hide. Extremely expensive.
 /datum/export/stack/skin/xeno
-	cost = 3000
+	cost = 500
 	unit_name = "alien hide"
 	export_types = list(/obj/item/stack/sheet/animalhide/xeno)
 
-
 // Common materials.
 // For base materials, see materials.dm
 
-// Plasteel. Lightweight, strong and contains some plasma too.
 /datum/export/stack/plasteel
-	cost = 305 // 2000u of plasma + 2000u of metal.
+	cost = 155 // 2000u of plasma + 2000u of metal.
 	message = "of plasteel"
 	export_types = list(/obj/item/stack/sheet/plasteel)
 
-// Reinforced Glass. Common building material. 1 glass + 0.5 metal, cost is rounded up.
+// 1 glass + 0.5 metal, cost is rounded up.
 /datum/export/stack/rglass
 	cost = 8
 	message = "of reinforced glass"
 	export_types = list(/obj/item/stack/sheet/rglass)
 
-// Bluespace Polycrystals. Uncommon.
 /datum/export/stack/bscrystal
-	cost = 750
+	cost = 300
 	message = "of bluespace crystals"
 	export_types = list(/obj/item/stack/sheet/bluespace_crystal)
 
-// Wood. Quite expensive in the grim and dark 26 century.
 /datum/export/stack/wood
-	cost = 50
+	cost = 30
 	unit_name = "wood plank"
 	export_types = list(/obj/item/stack/sheet/mineral/wood)
 
-// Cardboard. Cheap.
 /datum/export/stack/cardboard
 	cost = 2
 	message = "of cardboard"
 	export_types = list(/obj/item/stack/sheet/cardboard)
 
-// Sandstone. Literally dirt cheap.
 /datum/export/stack/sandstone
 	cost = 1
 	unit_name = "block"
 	message = "of sandstone"
 	export_types = list(/obj/item/stack/sheet/mineral/sandstone)
 
-// Cable.
 /datum/export/stack/cable
 	cost = 0.2
 	unit_name = "cable piece"
 	export_types = list(/obj/item/stack/cable_coil)
 
 // Weird Stuff
 
-// Alien Alloy. Like plasteel, but better.
-// Major players would pay a lot to get some, so you can get a lot of money from producing and selling those.
-// Just don't forget to fire all your production staff before the end of month.
 /datum/export/stack/abductor
 	cost = 5000
 	message = "of alien alloy"
 	export_types = list(/obj/item/stack/sheet/mineral/abductor)
 
-// Adamantine. Does not occur naurally.
 /datum/export/stack/adamantine
 	unit_name = "bar"
-	cost = 7500
+	cost = 10000
```

Comment:
Adamantine already makes anything else I'm aware of irrelevant as it can be mass-produced via stabilized metal/industrial adamantine/reproductive adamantine crossbreeds. While this does require some setup, it allows you to get hundreds of thousands of cargo points reasonably quickly. Decrease the price of adamantine if anything.